config/environments/production.rb

```diff
@@ -76,4 +76,7 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Twilio
+  config.middleware.use Rack::TwilioWebhookAuthentication, Rails.application.secrets.twilio_auth_token, '/voice'
```

Comment:
Line is too long. [112/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.